src/components/table-toolbar/KeycloakDataTable.tsx

```diff
@@ -215,24 +224,14 @@ export function KeycloakDataTable<T>({
             selected: !!selected.find(
               (v) => _.get(v, "id") === _.get(value, "id")
             ),
-            isOpen: false,
-            cells: columns.map((col) => {
-              if (col.cellRenderer) {
-                return { title: col.cellRenderer(value) };
-              }
-              return _.get(value, col.name);
-            }),
+            isOpen: detailColumns?.[0]?.enabled?.(value) ? false : undefined,
```

Comment:
If I get this correctly `isOpen` is `false` if `enabled()` is `true`? Otherwise it's also a falsy value, is that correct behavior? I was expecting something like:

```suggestion
            isOpen: detailColumns?.[0]?.enabled?.(value) ?? false,
```

Reply:
Since this logic is used twice in this function perhaps it can be extracted into a separate variable?

```ts
const isColumnEnabled = detailColumns?.[0]?.enabled?.(value) ?? false;
```

Reply:
No, it should only be `false` when "detailColumns" are not undefined and enabled for that value, but your right I can make it a function